dtsi.py

```diff
@@ -14,14 +18,52 @@ def generate_supplies(options: Options):
 	if options.regulator:
 		for r in options.regulator:
 			s += f"\t\t{r}-supply = <&...>;\n"
+		s += f"\n"
 	return s
 
 
 def generate_gpios(options: Options):
 	s = ""
 	for name, flags in options.gpios.items():
 		flags = "GPIO_ACTIVE_LOW" if flags & GpioFlag.ACTIVE_LOW else "GPIO_ACTIVE_HIGH"
-		s += f"\t\t{name}-gpios = <&msmgpio XY {flags}>;\n"
+		s += f"\t\t{name}-gpios = <&tlmm XY {flags}>;\n"
+
+	if has_gpio_backlight(options):
```

Comment:
I'd just inline this

```suggestion
	if 'backlight' in options.gpios:
```